warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -340,7 +340,11 @@ public int run(String[] args) throws Exception {
                 log.error(e);
                 log.info("Deleting orphaned directory: " + workDirPath);
                 try {
-                    inputFs.delete(workDirPath, true);
+                    if (writeDirectlyToDest) {
```

Comment:
Not sure why you need this if condition. Believe this can just be outputFs.delete(workDirPath, true) because if !writeDirectlyToDest then outputFs=inputFs.

Reply:
Yep that makes sense, thanks